File: PF_EVClass/MidTerm/findavg.py

def calc_avg(nList):
    sum = 0
    for n in nList:
        sum = sum + n

    res = sum / len(nList)
    return res

listNums = [8,8,8,8,8,8,8]
A = calc_avg(listNums)
print(A)

File: PF_EVClass/MidTerm/findavg.c
#include <stdio.h>



float calc_avg(int nList[7])
{
    int sum = 0;
    for(int i=0;i<7;i++)
        sum = sum + nList[i];

    float res = (float)sum / 7;
    return res;
}

int main()
{
    int listNums[7] = {8,8,8,8,8,7,8};
    float A = calc_avg(listNums);
    printf("%f\n",A);


    return 0;
}